Thread ticker + thread sur la main loop du serveur

aurelienCastellarnau opened the pull request:
Compile.

 - ticker threadé
- main loop threadée

ghost commented:
Attention aux fichiers temp poussés, sinon so far nothing to say.
Je me demande simplement à quel point c'est compliqué ou simple de faire des sleeps en ms.

aurelienCastellarnau commented:
Je regarde ca au plus vite

Commit:
feature: ticker thread + thead sur la main loop
Add: my_putstr pour éviter les complications avec write lors des exécutions de threads
add le thread du ticker dans thread.c (+ .h)
add un game_info.c pour déclarer un simili singleton
add la serialization de la game_info

Commit:
indent in main folder

Commit:
implémentaiton du ticker et de la serialization/deserialization de la game_info
